definitions/bulk-action/json/AuthorisationCaseState.json

```diff
@@ -82,5 +82,12 @@
     "CaseStateID": "DNPronounced",
     "UserRole": "caseworker-divorce-superuser",
     "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "09/11/2018",
+    "CaseTypeID": "DIVORCE_BulkAction",
+    "CaseStateID": "dropped",
```

Comment:
We usually us Captialized states, should we do that here too? i.e. `Dropped` just to help differentiate

Reply:
Also could be `DroppedBulkCase` - as it is more informative from the user's perspective

Reply:
ah yup, Capitalised for consistency, about the `DroppedBulkCase` do yo mean more informative for devs? this is the Id the users won't see this, and they already have the bulk case context by given by the case type

Reply:
Yeah for devs - I guess given the context it's ok